srv/SpawnEntity.srv

```diff
@@ -17,16 +17,30 @@ string uri                              # Resource such as SDFormat, URDF, USD o
 string resource_string                  # An entity definition file passed as a string.
                                         # Simulators may support spawning from a file generated on the fly (e.g. XACRO).
                                         # Check whether it is supported by your simulator through GetSimulatorFeatures.
+                                        # If uri field is not empty, resource_string field will be ignored.
 string entity_namespace                 # Spawn the entity with all its interfaces under this namespace.
-geometry_msgs/PoseStamped initial_pose  # Initial entity pose. Simulators will spawn the entity with its top-level link
-                                        # transform matching this pose.
+geometry_msgs/PoseStamped initial_pose  # Initial entity pose.
                                         # The header contains a reference frame, which defaults to global "world" frame.
                                         # This frame must be known to the simulator, e.g. of an object spawned earlier.
                                         # The timestamp field in the header is ignored.
 
 ---
 
-bool success                        # Return true if spawned successfully.
-string status_message               # On failure, a user-friendly error message.
-string entity_name                  # Spawned entity full name, guaranteed to be unique in the simulation.
-                                    # If allow_renaming is true, it may differ from the request name field.
+# Additional result.result_code values for this service. Check result.error_message for further details.
+uint8 NAME_NOT_UNIQUE       = 101       # Given name is already taken by entity and allow_renaming is false.
```

Comment:
Are those supposed to start from 101?

Reply:
Yes, same as with the other case - Result message defines generic ones, messages are to define >100 space specific ones, as indicated in comments.